packages/next/next-server/server/image-config.ts

```diff
@@ -21,4 +27,5 @@ export const imageConfigDefault: ImageConfig = {
   path: '/_next/image',
   loader: 'default',
   domains: [],
+  placeholder: PlaceholderValue.EMPTY,
```

Comment:
Is it ok that placeholder is currently getting a default value assigned twice? (Once in experimental and once in the image defaults?) I assume only one of them is actually getting used

Reply:
Yeah, this is because we're using an experimental flag as an image config. I've refactored the config name to better reflect the intent but let me know if I should just be using it directly instead.